modules/integration/src/test/scala/scala/cli/integration/TestUtil.scala

```diff
@@ -15,15 +15,24 @@ object TestUtil {
   val cliKind: String      = sys.props("test.scala-cli.kind")
   val isNativeCli: Boolean = cliKind.startsWith("native")
   val isCI: Boolean        = System.getenv("CI") != null
+  val isDebug: Boolean     = sys.props("test.scala-cli.debug.is-debug") == "true"
+  val debugPort: String    = sys.props("test.scala-cli.debug.port")
```

Comment:
Like above, something like `debugPortOpt: Option[String]` should be enough, no need of two variables.